modules/phase_field/include/ics/IsolatedBoundingBoxIC.h

```diff
@@ -0,0 +1,52 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "InitialCondition.h"
+
+// Forward Declarations
+class IsolatedBoundingBoxIC;
+
+template <>
+InputParameters validParams<IsolatedBoundingBoxIC>();
+
+/**
+ * MultiBoundingBoxIC allows setting the initial condition of a value of a field inside and outside
+ * multiple bounding boxes. Each box is axis-aligned and is specified by passing in the x,y,z
+ * coordinates of opposite corners. Separate values for each box may be supplied.
+ */
+class IsolatedBoundingBoxIC : public InitialCondition
```

Comment:
So is there any particular reason that we don't just enhance MultiBoundingBoxIC versus adding this completely new IC? It looks like you may have started from MultiBoundingBoxIC as the comment above is copied from that class. These appear very close. At the very least, this class should probably inherit from MultiBoundingBoxIC so that you can reuse most of the parameters.

Reply:
To enhance MultiBoundingBoxIC and add the int_width option, it should be able to tell whether the boxes defined by the user are nested, isolated, or partially overlapping, or a mix of these in 2D and 3D, in order to interpolate across boundaries. I haven't figure out an easy way to do so and the algorithm might get overly complicated. Specifying nested or isolated boxes is much easier. 
MultiBoundingBoxIC is also here to keep because it allows the mix of nested and isolated boxes (if necessary), though without interfacial width option.

Reply:
OK - so I can see your point here. You still may want to consider inheriting from the base though since the names and types of your input parameters are the same and you could then factor out the sharp interface code so it only appears once.

BTW - I won't force you to do all this as your contribution is good. My job is to look for ways to reduce code and remove duplication so I'll always suggest it.

Reply:
I changed the header files of the two new ICs to inherit from the current MultiBoundingBoxIC header file.